http-server/src/main/java/org/triplea/server/lobby/game/listing/GameListingController.java

```diff
@@ -0,0 +1,115 @@
+package org.triplea.server.lobby.game.listing;
+
+import com.google.common.annotations.VisibleForTesting;
+import es.moki.ratelimij.dropwizard.annotation.Rate;
+import es.moki.ratelimij.dropwizard.annotation.RateLimited;
+import es.moki.ratelimij.dropwizard.filter.KeyPart;
+import io.dropwizard.auth.Auth;
+import java.util.Collection;
+import java.util.concurrent.TimeUnit;
+import javax.annotation.security.RolesAllowed;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import org.triplea.http.client.lobby.game.listing.GameListingClient;
+import org.triplea.http.client.lobby.game.listing.LobbyGame;
+import org.triplea.http.client.lobby.game.listing.LobbyGameListing;
+import org.triplea.http.client.lobby.game.listing.UpdateGameRequest;
+import org.triplea.lobby.server.db.data.UserRole;
+import org.triplea.server.access.AuthenticatedUser;
+
+/** Controller with endpoints for posting, getting and removing games. */
+@Builder
+@Produces(MediaType.APPLICATION_JSON)
+@Path("/")
+@AllArgsConstructor(
+    access = AccessLevel.PACKAGE,
+    onConstructor_ = {@VisibleForTesting})
+@RolesAllowed(UserRole.ANONYMOUS)
```

Comment:
Can you clarify what `@RolesAllowed(UserRole.ANONYMOUS)` is indicating here?

Reply:
It means you need to be authenticated to access the endpoint. Anonymous is lowest authentication level, so anyone that is authenticated can access. It's analogous to an "unquarantined" socket connection. In other words, users that are not banned.

Reply:
Ok. It reads a little odd but I see what you mean. I think its just sounds a bit odd to have an "anonymous" role and that all the other user types would be able to authenticate to be "anonymous" role. Where as user vs mod vs admin is a more natural progression. But that might just be the reality of how its structured.